addon-api/background/MessagePasser.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Handles message passing for background scripts.
+ */
+
+export default class MessagePasser extends EventTarget {
+  constructor(addonObject) {
+    super();
+    this._addonId = addonObject.self.id;
+  }
```

Comment:
Probably no use to extend EventTarget if we dont take advantage of their APIs
```suggestion
/**
 * Handles message passing for background scripts.
 */

export default class MessagePasser {
  constructor(addonObject) {
    this._addonId = addonObject.self.id;
  }
```

Reply:
consistency

Reply:
This should actually be extending listenable so the bg can access it in scratchAddons to be consistent with userscripts.